evalml/tests/objective_tests/test_objectives.py

```diff
@@ -189,3 +190,25 @@ def test_is_defined_for_problem_type():
 @pytest.mark.parametrize("obj", _all_objectives_dict().values())
 def test_get_objectives_all_expected_ranges(obj):
     assert len(obj.expected_range) == 2
+
+
+@pytest.mark.parametrize(
+    "nullable_ltype",
+    ["BooleanNullable", "IntegerNullable", "AgeNullable"],
+)
+def test_binary_objective_handle_nullable_types(nullable_type_target, nullable_ltype):
+    y_true = nullable_type_target(ltype=nullable_ltype, has_nans=False)
+
+    obj = CustomBinaryClassificationObjective()
+    y_true_d = obj._handle_nullable_types(y_true)
+    assert str(y_true_d.ww.logical_type) != nullable_ltype
+
+
+@pytest.mark.parametrize("data_type", ["np", "pd"])
+def test_binary_objective_handle_nullable_types_non_ww_input(make_data_type, data_type):
```

Comment:
I was considering this, and we have ways of setting IntegerNullable and BooleanNullalbe via `make_data_type`, but there's not a great way to set AgeNullable without making larger changes to the fixture (it does a ww.init_series call, which would remove the AgeNullable logical type), and I'm worried it'd cause a bunch of unrelated changes. 

I think also with a change from a comment above to also pass pandas series without woodwork types into the downcast util, this will change these tests anyway, so I'll figure out how we can parametrize with all three `data_type`s even if it doesn't make sense to use `make_data_type` with all three of them.

Reply:
After the changes to allow pandas series without woodwork info to be passed into the downcast util (e5e7a529b), I still feel like the tests are different enough that it doesn't make sense to combine them.